modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/EventUtils.java

```diff
@@ -202,6 +202,15 @@ public static MetadataCollection getEventMetadata(Event event, EventCatalogUIAda
       metadata.addField(newUID);
     }
 
+    if (metadata.getOutputFields().containsKey(DublinCore.PROPERTY_PUBLISHER.getLocalName())) {
+      MetadataField<?> publisher = metadata.getOutputFields().get(DublinCore.PROPERTY_PUBLISHER.getLocalName());
+      metadata.removeField(publisher);
+      MetadataField<String> newPublisher = MetadataUtils.copyMetadataField(publisher);
+      newPublisher.setValue(event.getPublisher());
+      newPublisher.setReadOnly(true);
```

Comment:
Why are you setting read-only to true here instead of using the read-only value configured? Is that because you want it to be editable on event-creation, but read-only later? Then I feel like it would be better to introduce a second configuration value for this since it's rather unintuitive that the read-only value can be configured, but is then not obeyed as expected. The metada fields are messy enough as it is...

Reply:
@KatrinIhler,

1. _"Why are you setting read-only to true here instead of using the read-only value configured? Is that because you want it to be editable on event-creation, but read-only later?"_
Yes.

2. _"Then I feel like it would be better to introduce a second configuration value for this since it's rather unintuitive that the read-only value can be configured, but is then not obeyed as expected"_
Much better, but in my opinion this is an important change that I should not decide it. All fields use the same configuration properties: _inputID_, _outputID_, _label_, _type_, _readOnly_, _required_ and _order_.
I have not seen the impact of a new property, but I am sure, it is not trivial, and only for the publisher...
Besides, if it were editable, the list would not show the logged in user. It is not easy. The code is in an utility class and we have no access to the service _SecurityService_. In _Series_ is diferent. _Events_ and _Series_ do not resolve the same problems in the same way.

I see two solutions:
1. The easiest solution would be to remove that line:
newPublisher.setReadOnly(true);
and the property would be also editable (without the logged in user).
2. The other, also very easy, to add the line:
publisher.setReadOnly(false);
when we create a new event, and delete the property in the file configuration. But I do not like this.

Reply:
@jchssystems Yeah, there's probably no perfect solution for this right now. :/ I would prefer solution 1 because while it's not exactly what we want, it doesn't make the metadata code more complicated and obscure than it already is.